test/powershell/Modules/Microsoft.PowerShell.Utility/Remove-Variable.Tests.ps1

```diff
@@ -5,7 +5,7 @@ Describe "Remove-Variable" -Tags "CI" {
     It "Should throw an error when a dollar sign is used in the variable name place" {
 	New-Variable -Name var1 -Value 4
 
-	Remove-Variable $var1 -ErrorAction SilentlyContinue | Should Throw
+	{ Remove-Variable $var1 -ErrorAction Stop } | Should Throw
```

Comment:
Wouldn't it be better to change this to:

```powershell
Remove-Variable $var1 -ErrorAction SilentlyContinue -ErrorVariable err
$err.FullyQualifiedErrorId | Should Be "VariableNotFOund,Microsoft.PowerShell.Commands.RemoveVariableCommand"
```

(applies to all the `Should throw` checks)

Reply:
I'll update the changes that I made to get it to work in this way, but I will defer changing all of these to a follow-on PR. My plan is to not update all tests with this PR

Reply:
Ok.  Closed.